addons/mail/static/src/models/message/message.js

```diff
@@ -671,6 +691,9 @@ function factory(dependencies) {
         isEmpty: attr({
             compute: '_computeIsEmpty',
         }),
+        isBodyEmpty: attr({
+            compute: '_computeIsBodyEmpty',
+        }),
```

Comment:
Small docstring + move above `isBodyEqualSubtypeDescription`.
_(small docstring will clear all confusion in how to read this field, if there was any confusion)_

Reply:
Done !
Not sure you ll be happy whit the comment's since this is a very specific field.